package.json

```diff
@@ -102,7 +102,6 @@
     "node-fetch": "^2.6.7",
     "outvariant": "^1.3.0",
     "path-to-regexp": "^6.2.0",
-    "statuses": "^2.0.0",
```

Comment:
removing it from `dependencies`

it's already in devDependencies, and if it's not in `dependencies` then tsup will inline it at build-time

Reply:
Huh, that's funny how it was in both deps and dev deps. A good find.